fms/utils/fusion.py

```diff
@@ -0,0 +1,41 @@
+import torch
+import torch.nn as nn
+
+
+def _fusion_converted(module: nn.Module, fuse: bool):
+    if not fuse and hasattr(module, "unfuse"):
+        result = module.unfuse()
+        del module
+    elif fuse and hasattr(module, "fuse"):
```

Comment:
hmm... I don't love this, 'fuse' cane be an overloaded term that has lots of uses (i.e. during compile not just k/q/v). It might be cleaner to have this be explicit on the relevant modules we know we want to fuse, and more of a functional rather than OO api

Reply:
I was thinking of renaming this to something like "fuse_weights", since we will want this in other modules like mlp GLU. If we have modules in the future as well we want to fuse/unfuse we can then use that (similar to how we did tp with `to_tp`)